src/modules/nunjucks/DateFilter.ts

```diff
@@ -0,0 +1,27 @@
+export function dateFilter(value: string ): string {
```

Comment:
Unnecessary whitespace before closing bracket.

Reply:
There should also be validation if string value does not conform ISO format. Regex could be used.

Reply:
Regex would definitely be neater, thanks for the suggestion.